src/types/field/attachmentField.ts

```diff
@@ -1,18 +1,14 @@
-import { IField, IFieldSchema } from './baseField'
+import {
+  AttachmentFieldBase,
+  AttachmentSize,
+  BasicField,
+} from '../../../shared/types/field'
 
-export enum AttachmentSize {
-  OneMb = '1',
-  TwoMb = '2',
-  ThreeMb = '3',
-  SevenMb = '7',
-  TenMb = '10',
-  TwentyMb = '20',
-}
+import { IFieldSchema } from './baseField'
 
-export interface IAttachmentField extends IField {
-  attachmentSize: AttachmentSize
-}
+export { AttachmentSize }
```

Comment:
is this really necessary? seems to be complicating our import/exports

Reply:
This is done to keep all imports at parity, since many `AttachmentSize` imports are imported from this file in the backend. As above, I can re-export everything via a central location in `src/types` and correct all import paths if desired